BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.m

```diff
@@ -173,11 +174,14 @@ - (instancetype)init
         // manager uses the session as a lock object when enqueuing operations.
         _queueManager = [[BOXParallelAPIQueueManager alloc] init];
 
+        _urlSessionManager = [[BOXNSURLSessionManager alloc] init];
```

Comment:
BOXParallelAPIQueueManager is for ensure ordering of requests, BOXNSURLSessionManager is a wrap around for NSURLSession which creates actual NSURLSessionTask to be executed in request.